showcase/src/app/news/news-component/news.component.html

```diff
@@ -1,21 +1,20 @@
-<h1 class="mat-display-3" translate>Stark news</h1>
-<h2 class="mat-display-1">Discover here the latest news about the Stark framework!</h2>
+<div class="mat-display-3">Stark news</div>
+<div class="mat-display-1">Discover here the latest news about the Stark framework!</div>
 
-<news-item [release]="'1.1.0'" [newsDate]="'19/07/2018'">
-	<div class="news-item-title">Stark News page is finally here! </div>
+<news-item [release]="'1.0.0'" [newsDate]="'27/09/2018'">
+	<div class="news-image second-news-image"></div>
+	<div class="news-item-title">A very happy occasion</div>
 	<div class="news-item-content">
-		The news page is there, get happy! <br>
-		Below is an image of this first news displayed as a full example of the news-item component <br>
-		<img class="first-news-image">
+		Stark Logo was born this week! <br>
```

Comment:
Let's try to avoid using `<br>` and use `<p>` instead, and set any custom spacing via CSS... this way is more flexible and easier to change ;)